docs/admin/addons.md

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
I'd love to see all addons move to being `kubectl apply -f`.  It'll be much easier to install and manage.  They were done the way they were done because we didn't have daemonsets at the time, etc.